packages/sitecore-jss-nextjs/src/editing/editing-render-middleware.test.ts

```diff
@@ -70,9 +70,6 @@ const mockResponse = () => {
   res.clearPreviewData = spy(() => {
     return res;
   });
-  res.setPreviewData = spy(() => {
```

Comment:
This spy is still needed

Reply:
Yeah yeah i accidently removed the `setPreviewData`.
I wanted to remove the `clearPreviewData`.